packages/gatsby-theme-newrelic/src/utils/localization.js

```diff
@@ -5,6 +5,10 @@ export const localizePath = ({ path, locale }) => {
     return path;
   }
 
+  if (path === '/') {
```

Comment:
It seems this is already handled by the logic on line 14. I can spin up the theme on develop, switch to Japanese, and get the home link as I would expect. I also see this behavior on the current docs site if I switch to JP and click on the logo. Maybe I'm missing something?

Reply:
Yeah that's what I thought so too, but I didn't understand why I was seeing the behavior I was seeing so I added an extra conditional. When you go directly to docs.newrelic.com/jp and click on the footer logo it takes you to the english homepage.

Reply:
Actually I'm seeing it even when I switch to japanese while I'm on the homepage. it's only in the footer logo 🤔 . Maybe it's because we shadow that logo?

Reply:
Ah ha! Thats the ticket. It looks like we are importing `Link` from `gatsby`, not our custom `Link` component:

https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/GlobalFooter.js#L6

Switch that import and I bet that fixes this issue.

Reply:
sweeet! will make that change

Reply:
all changed :)